bitnami/influxdb/templates/influxdb/standalone-dpl.yaml

```diff
@@ -0,0 +1,253 @@
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "influxdb.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: influxdb
+  template:
+    metadata:
+      labels: {{- include "influxdb.labels" . | nindent 8 }}
+        app.kubernetes.io/component: influxdb
+    spec:
+{{- include "influxdb.imagePullSecrets" . | nindent 6 }}
+      affinity:
+        {{- if eq .Values.influxdb.antiAffinity "hard" }}
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - topologyKey: "kubernetes.io/hostname"
+              labelSelector:
+                matchLabels: {{- include "influxdb.matchLabels" . | nindent 20 }}
+                  app.kubernetes.io/component: influxdb
+        {{- else if eq .Values.influxdb.antiAffinity "soft" }}
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              podAffinityTerm:
+                topologyKey: kubernetes.io/hostname
+                labelSelector:
+                  matchLabels: {{- include "influxdb.matchLabels" . | nindent 20 }}
+                    app.kubernetes.io/component: influxdb
+        {{- end }}
+        {{- if .Values.influxdb.nodeAffinity }}
+        nodeAffinity: {{- toYaml .Values.influxdb.nodeAffinity | nindent 10 }}
+        {{- end }}
+      {{- with .Values.influxdb.nodeSelector }}
+      nodeSelector: {{- tpl (toYaml .) $ | nindent 8 }}
+      {{- end }}
+      {{- with .Values.influxdb.tolerations }}
+      tolerations: {{- tpl (toYaml .) $ | nindent 8 }}
+      {{- end }}
+      {{- if .Values.influxdb.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.influxdb.securityContext.fsGroup }}
+      {{- end }}
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled }}
+      initContainers:
+        - name: init-chmod-data
+          image: {{ include "influxdb.volumePermissions.image" . }}
+          imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
+          command:
+            - sh
+            - -c
+            - |
+              mkdir -p /bitnami/influxdb/{data,meta,wal}
+              chown -R "{{ .Values.influxdb.securityContext.runAsUser }}:{{ .Values.influxdb.securityContext.fsGroup }}" /bitnami/influxdb/{data,meta,wal}
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: data
+              mountPath: /bitnami/influxdb
+        {{- end }}
+      containers:
+        - name: influxdb
+          image: {{ include "influxdb.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          {{- if .Values.influxdb.securityContext.enabled }}
+          securityContext:
+            runAsUser: {{ .Values.influxdb.securityContext.runAsUser }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.image.debug | quote }}
```

Comment:
Why is this a ternary and not just `.Values.image.debug`? Doesn't seem consistent with other values. E.g.

```yaml
- name: INFLUXDB_HTTP_AUTH_ENABLED
  value: {{ .Values.authEnabled | quote }}
```

Reply:
This "ternary" way also covers the case where `.Values.image.debug` is not defined